### ISSUE TITLE
subscribe/unsubscribe to all statuses

### ISSUE THREAD

**Issue by emiltin** [rsmp-nordic/rsmp_core#12]
A way to subscribe and unsubscribe to all statuses at once

**Comment by otterdahl** [comment]
Good idea. But there are edge cases that may cause problems. These needs to be identified first.

For instance, if the assumption is that update on change is active for all statuses (uRt=0) then traffic data won't work correctly.

**Comment by emiltin** [comment]
Yeah I think there could be several edge cases.

I was initially thinking the message would be useful mainly in development settings, but I understand that in Sweden you subscribe to all statuses of traffic light controllers, and in such cases it would probably be useful to be able to send a single message to subuscribe to everything.

Techically, traffic data does work with interval=0, it's just that it's often impractical to receive a message for every cyclist or car counted. But sometimes it could be useful.

We could allow setting the interval when subscribing to all statuses. But the issue seems to be that you want to use different intervals for different statues.

Maybe provide a default, and then settings for particular statuses?
Eg. default=60, S0201=60, S023=off

**Comment by emiltin** [comment]
or initialize just support unsubscribing from all with a single short command. currently i see e.g. MobiMaestro sending a sequence of unsubscribe messages each time a tlc connects.

**Comment by emiltin** [comment]
the main benefit would be to reduce bandwidth in cases where you want all data from a device. instead of sending a lot of different status messages, you send a single message.
so it's mainly related to performance, and to some degree to convenience.

for unsubscribring, it's about convenience, plus it makes it easier to ensure that you really unsubscribe to everything.
